man/machinery-inspect.1.md

```diff
@@ -68,6 +68,8 @@ trigger errors.
 
       $ `machinery` inspect --skip-files=/file\\,with_comma myhost
 
+    Note: Paths are not expanded.
```

Comment:
We should make it clear that this is only true for the file or directory names which are given as argument, not if a file is given with the `@` prefix. So maybe saying "file or directory names" instead of paths (as it is done in the documentation in other places as well) would be better. And maybe it would make it more clear what is meant by providing an example (e.g. `../path`).